### PR TITLE
Dynamic forms

### PR DESCRIPTION
**Pull request by jamessimone**
Addresses #41. Can't yet delete page layouts because at least one per object is required.

_It's just a biiiiiiig old blob of XML chaaaaanges_

### CODE REVIEW

File: nebula-logger/main/logger/objects/LogEntry__c/fields/HasException__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jongpie) [inline nebula-logger/main/logger/objects/LogEntry__c/fields/HasException__c.field-meta.xml:1]: I just remembered that we might be able to do the conditional rendering in the flexipage without a new field - you could use the picklist field `LogEntry__c.Type__c == 'Exception'` to get essentially the same result. But let me know if you think it's still worth keeping the new field.

Comment (jamessimone) [reply nebula-logger/main/logger/objects/LogEntry__c/fields/HasException__c.field-meta.xml:1]: I think we should keep it to make things consistent in terms of how the flexipage is filtered - reduces the cognitive overhead.

Comment (jongpie) [reply nebula-logger/main/logger/objects/LogEntry__c/fields/HasException__c.field-meta.xml:1]: Works for me!

**Comment by jongpie** [comment]
I deployed it to my sandbox to review - I think this looks good! Couple of thoughts -
- Feel free to merge (since it's being merged with your `winter21` branch instead of `master`)
- After this gets merged to `master`, I'll probably do another round of updates on the flexipages to add some of the new fields from #44 and #57 
- For now, I'm planning to maintain both the old page layout & the new dynamic flexipage layout - I'd like to keep everything working in Classic UI for a bit longer (which doesn't use flexipages at all). At some point, I might drop support for Classic UI/stop worrying about the page layout, but I'll keep it updated for the time being

**Comment by jamessimone** [comment]
Sounds good, thanks man!

**Pull request merged**